app/helpers/application_helper/toolbar/chargeback_report_center.rb

```diff
@@ -0,0 +1,44 @@
+class ApplicationHelper::Toolbar::ChargebackReportCenter < ApplicationHelper::Toolbar::Basic
+  button_group('chargeback_reports_download_main', [
+    select(
+      :chargeback_reports_download_choice,
+      'fa fa-download fa-lg',
+      N_('Download'),
+      nil,
+      :klass => ApplicationHelper::Button::ChargebackReportDownloadChoice,
+      :items => [
+        button(
+          :chargeback_reports_download_text,
+          'fa fa-file-text-o fa-lg',
+          N_('Download this report in text format'),
+          N_('Download as Text'),
+          :klass => ApplicationHelper::Button::ChargebackReportDownload,
+          :url   => "/render_txt"),
```

Comment:
Here you can specify the RBAC feature somehow.

Reply:
Also on this one, i don't think there is a way to do a RBAC check in place.

Reply:
Right, I tagged the wrong line :laughing: sorry